src/Middleware/Session/src/CookieProtection.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics.CodeAnalysis;
```

Comment:
@blowdart This is problematic and we don't have a trimmable approach for doing something like data protection as yet. Something to think about in the future.